examples/camera_manager/infos/camera_info.xml

```diff
@@ -0,0 +1,664 @@
+                            <roption name="1/4000" value="0.00025" />
+                            <roption name="1/8000" value="0.000125" />
+                        </parameterrange>
+                    </parameterranges>
+                </option>
+                <option name="2720 x 1530 48fps (UHD)" value="13">
+                    <parameterranges>
+                        <parameterrange parameter="CAM_SHUTTERSPD" condition="CAM_MODE=1 AND CAM_EXPMODE=1">
+                            <roption name="1/60" value="0.016666" />
+                            <roption name="1/125" value="0.008" />
+                            <roption name="1/250" value="0.004" />
+                            <roption name="1/500" value="0.002" />
+                            <roption name="1/1000" value="0.001" />
+                            <roption name="1/2000" value="0.0005" />
+                            <roption name="1/4000" value="0.00025" />
+                            <roption name="1/8000" value="0.000125" />
+                        </parameterrange>
+                    </parameterranges>
+                </option>
+                <option name="2720 x 1530 30fps (UHD)" value="14">
+                    <parameterranges>
+                        <parameterrange parameter="CAM_SHUTTERSPD" condition="CAM_MODE=1 AND CAM_EXPMODE=1">
+                            <roption name="1/30" value="0.033333" />
+                            <roption name="1/60" value="0.016666" />
+                            <roption name="1/125" value="0.008" />
+                            <roption name="1/250" value="0.004" />
+                            <roption name="1/500" value="0.002" />
+                            <roption name="1/1000" value="0.001" />
+                            <roption name="1/2000" value="0.0005" />
+                            <roption name="1/4000" value="0.00025" />
+                            <roption name="1/8000" value="0.000125" />
+                        </parameterrange>
+                    </parameterranges>
+                </option>
+                <option name="2720 x 1530 24fps (UHD)" value="15">
+                    <parameterranges>
+                        <parameterrange parameter="CAM_SHUTTERSPD" condition="CAM_MODE=1 AND CAM_EXPMODE=1">
+                            <roption name="1/30" value="0.033333" />
+                            <roption name="1/60" value="0.016666" />
+                            <roption name="1/125" value="0.008" />
+                            <roption name="1/250" value="0.004" />
+                            <roption name="1/500" value="0.002" />
+                            <roption name="1/1000" value="0.001" />
+                            <roption name="1/2000" value="0.0005" />
+                            <roption name="1/4000" value="0.00025" />
+                            <roption name="1/8000" value="0.000125" />
+                        </parameterrange>
```

Comment:
Do you need all these video formats just for an example? I would suggest to simplify a bit.

Reply:
It's a one-to-one copy from https://mavlink.io/en/services/camera_def.html#full_example

Reply:
Ok, fair enough.